git-commitgraph/tests/fixtures/octopus_merges.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -eu -o pipefail
+
+export GIT_AUTHOR_DATE="2000-01-01 00:00:00 +0000"
+export GIT_AUTHOR_EMAIL=author@example.com
+export GIT_AUTHOR_NAME=author
+export GIT_COMMITTER_DATE="2000-01-02 00:00:00 +0000"
+export GIT_COMMITTER_EMAIL=committer@example.com
+export GIT_COMMITTER_NAME=committer
+
+mkdir -p "$1"
```

Comment:
It really is wonderful to have a few shell scripts that use git itself to generate commit-graphs for common cases.

Reply:
I committed the generated repos directly just to ensure we had v1 fixtures available even if git moved to v2. If we shouldn't worry about that yet, I'd just as soon have the tests run these scripts every time instead of pre-generating them.

Reply:
Thank you! I wonder if there is a way to keep the generator scripts around, since it's usually a good idea to document where fixtures are coming from if feasible. The makefile has a [maintenance section](https://github.com/Byron/gitoxide/blob/791c05e08d9c6148fa0f89894c9b2abdadf3f503/Makefile#L158-L157), some of whose targets already regenerate fixtures. That should be a good fit.

Reply:
I meant I plan on keeping the `tests/fixtures/*.sh` scripts around but not commit their generated data to `tests/fixtures/v1`. The tests would run these scripts on demand.

Reply:
That sounds good, great :).